### ISSUE TITLE
Flake8 No Issues but Lint Check Reports Failure

### ISSUE THREAD

**Issue by lucasantarella** [wearerequired/lint-action#644]
Hi All,

Great library, I use it on a number of projects.

Recently ran into an issue with Flake8 and this action. When there are no issues reported from Flake8, the action still reports a failed check to Github Actions, making this difficult to use in branch protection rules and pull requests.

Here's an example:
<img width="1362" alt="image" src="https://user-images.githubusercontent.com/1783707/236516281-43dc0bdd-65c2-4106-b9eb-d939acbb75c3.png">

Full Log Archive:
[logs_7.zip](https://github.com/wearerequired/lint-action/files/11408395/logs_7.zip)

Github Actions YAML:
```yaml
name: Lint

on:
  push:
    branches:
      - master
      - develop
  pull_request:
    branches:
      - master
      - develop

jobs:
  run-linters:
    name: Run linters
    runs-on: ubuntu-latest
    permissions:
      checks: write
      contents: write
      pull-requests: write
    steps:
      - name: Check out Git repository
        uses: actions/checkout@v3
      - name: Set up Python
        uses: actions/setup-python@v4
        with:
          python-version: '3.10'
      - name: Install Python dependencies
        run: pip install black flake8
      - name: Run linters
        uses: wearerequired/lint-action@v2
        with:
          black: true
          black_args: --line-length 88
          flake8: true
          flake8_args: --config ./setup.cfg
```

Any help is appreciated.
